include/SDL3/SDL_gamepad.h

```diff
@@ -117,6 +117,7 @@ typedef enum SDL_GamepadType
     SDL_GAMEPAD_TYPE_PS3,
     SDL_GAMEPAD_TYPE_PS4,
     SDL_GAMEPAD_TYPE_PS5,
+    SDL_GAMEPAD_TYPE_PSMOVE,
```

Comment:
Inserting here changes enumerated values after it, thus breaking ABI. This should be inserted just before `SDL_GAMEPAD_TYPE_COUNT`